exercises/darts/cases_test.go

```diff
@@ -0,0 +1,73 @@
+package darts
+
+// Source: exercism/problem-specifications
+// Commit: efcb2c8 Darts: Flatten sub-cases to single level - to simplify code generation (#1510)
+// Problem Specifications Version: 2.0.0
+
+var testCases = []struct {
+	description string
+	x           float64
+	y           float64
+	expected    int
+}{
+	{
+		description: "A dart lands outside the target",
+		x:           -9.0,
+		y:           9.0,
+		expected:    0,
+	},
+	{
+		description: "A dart lands just in the border of the target",
+		x:           0.0,
+		y:           10.0,
+		expected:    1,
```

Comment:
I think we need to clarify whether we're using what's called _inward scoring_ (touching a score line earns the higher score) or _outward scoring_ (touching a score line earns the lower score). Both methods are commonly used in shooting sports. Here, the test data suggests inward scoring (if you clip the 1-point line, you get 1 point).

A real dartboard avoids this issue by using lines made of wire; the dart goes one side or the other!

Reply:
Good point.  A real dartboard is different.

I could mention the inward scoring in a hints.md that gets brought into the README.md from configlet run.  Unless maybe this is a case for clarification in the problem-specifications description.md, helping all tracks.